components/radio-button-group/__tests__/__snapshots__/radio-button-group.snapshot-test.jsx.snap

```diff
@@ -204,7 +210,8 @@ exports[`Radio Button Group Disabled DOM Snapshot 1`] = `
         >
           <input
             aria-describedby={undefined}
-            checked={false}
+            checked={true}
```

Comment:
I'm not sure why but this changed from false to true.

Reply:
Im not sure neither, is there any reason this would get updated?

Reply:
@interactivellama the checkbox was being forced as controlled component even when `checked` was not given

Reply:
Yeah, it's like removing the `!this.props.disabled` caused this, but it seems appropriate to allow disabled options that are selected